app-tracking-protection/vpn-internal/src/main/java/com/duckduckgo/vpn/internal/feature/rules/ExceptionRulesDebugReceiver.kt

```diff
@@ -87,7 +88,7 @@ class ExceptionRulesDebugReceiverRegister @Inject constructor(
 
     override fun onVpnStarted(coroutineScope: CoroutineScope) {
         // only for debug builds
-        if (appBuildConfig.isDebug) {
+        if (appBuildConfig.isDebug || appBuildConfig.isInternalBuild()) {
```

Comment:
This change may have an impact on https://app.asana.com/0/1202279501986195/1203536459696139